openquake/hmtk/plotting/seismicity/catalogue_plots.py

```diff
@@ -90,11 +88,11 @@ def plot_depth_histogram(
         To sample depth uncertainty choose number of samples
     """
     if ax is None:
-        fig, ax = plt.subplots(figsize=DEFAULT_SIZE)
+        fig, ax = plt.subplots(figsize=figure_size)
     else:
         fig = ax.get_figure()
     # Create depth range
-    if len(catalogue.data['depth']) == 0:
+    if len(catalogue.data['depth']) == 0:  # pylint: disable=len-as-condition
```

Comment:
You can just switch this to `if not len(...)`, there is no need to disable pylint's check for this